frontends/verilog/verilog_parser.y

```diff
@@ -871,6 +872,10 @@ task_func_decl:
 	};
 
 func_return_type:
+	hierarchical_type_id {
+		$$ = new AstNode(AST_WIRETYPE);
+		$$->str = *$1;
```

Comment:
I think this is missing a `delete $1`?

Reply:
Sure, I've added this. This `delete` is also missing from the `wire_type` production, so you may be interested in adding that to your other PR.